packer.mk

```diff
@@ -67,7 +67,15 @@ endif
 
 build-packer-windows:
 ifeq (${MODE},sigMode)
+ifeq (${HYPERV_GENRATION},V1)
 	@echo "${MODE}: Building with Hyper-v generation 1 VM and save to Shared Image Gallery"
+else
+ifeq (${GEN2_SIG_FOR_PRODUCTION},True)
```

Comment:
indent?

Reply:
`ifeq` is makefile syntax and cannot be preceded by TAB according to the explanation [here](https://stackoverflow.com/questions/28712585/when-to-use-space-or-tab-in-makefile)? 

Adding indentation fails [here](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=59467327&view=logs&j=05b0285b-7424-5da1-4fe4-29b48f7b1d3a&t=4f5de716-8c9d-57d8-0c59-15d673379c58). 
`-c: line 0: syntax error near unexpected token `,V1'
`/bin/bash: -c: line 0: `ifeq (,V1)'`

See the `"build-packer"` block in this file?